work2.py

```diff
@@ -0,0 +1,11 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+
+print('Введите трёхзначное число')
+a = int(input())
```

Comment:
Функцию `print` можно не использовать, потому что обычно приглашение к вводу пишется внутри функции `input`.
```suggestion
print('Введите трёхзначное число')
a = int(input('Введите трёхзначное число'))
```